File: src/sdk-reference/csharp/1/collection/search-specifications/snippets/search-specifications.cs
try {
  QueryOptions options;
  options.from = 0;
  options.size = 2;

  SearchResult* response = kuzzle.collection.searchSpecifications(R"({
    "query": {
      "match_all": {}
    }
  })", options);

  Console.WriteLine("Successfully retrieved " + response.fetched + " specifications");
} catch (KuzzleException e) {
  Console.Error.WriteLine(e.getMessage());
}

File: src/sdk-reference/cpp/1/collection/search-specifications/snippets/search-specifications.cpp
try {
  kuzzleio::query_options options;
  options.from = 0;
  options.size = 2;

  kuzzleio::SearchResult* response = kuzzle->collection->searchSpecifications(R"({
    "query": {
      "match_all": {}
    }
  })", &options);

  std::cout << "Successfully retrieved " << response->fetched << " specifications" << std::endl;
} catch (kuzzleio::KuzzleException &e) {
  std::cerr << e.what() << std::endl;
}
